cdap-common/src/main/java/co/cask/cdap/common/conf/Configuration.java

```diff
@@ -19,6 +19,7 @@
 import co.cask.cdap.common.utils.DirUtils;
 import com.google.common.base.Preconditions;
 import com.google.gson.stream.JsonWriter;
+import org.apache.hadoop.classification.InterfaceStability;
```

Comment:
This is an internal class, ok not to have annotation like this. If needed, use `@Beta`.